config/routes.rb

```diff
@@ -186,9 +186,44 @@
 
   # lectures routes
 
-  get "lectures/:id/food",
+  get "lectures/:id/material",
       to: "media#index",
-      as: "lecture_food"
+      as: "lecture_material"
+
+  # New semantic routes replace the old food routes
```

Comment:
This is indeed so much better than `?project=sesam` beforehand 😅, now its `/worked_examples` instead.
In the `application_helper.rb`, we have two more projects that are not listed here: `questions` and `remarks`. Are they handled differently?

Reply:
to the function `media_types` and `media_sorts`, which lists more media types than handled here, which is why I'm asking about `questions` and `remarks`

Reply:
The reason is that `question` and `remarks`, although they can be associated to a lecture, do not have an own show action that can be accessed from lecture level e.g. from the sidebar. But they are still needed here because they appear e.g. as fields in the media search or in the editor's view for lecture/course media.